guides/source/getting_started.md

```diff
@@ -2338,6 +2338,7 @@ Notifications module.
 class Product < ApplicationRecord
   include Notifications
 
+  has_many :subscribers, dependent: :destroy
```

Comment:
Isn't this part of `include Notifications`? It seems wrong to me to define the association twice

Reply:
Yes, thanks! 🙃